src/pages/Animetoast/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+
+export const Animetoast: pageInterface = {
+  name: 'Animetoast',
+  domain: 'https://www.animetoast.cc',
+  languages: ['German'],
+  type: 'anime',
+  isSyncPage(url: string): boolean {
+    return $('.entry-title.light-title').text() != null && url.includes('link=');
+  },
+  isOverviewPage(url: string): boolean {
+    return $('.entry-title.light-title').text() != null && !url.includes('link=');
+  },
+  sync: {
+    getTitle(url: string): string {
+      return $('.entry-title.light-title')
+        .text()
+        .replace(/(Ger Sub|Ger Dub)/gi, '')
+        .trim();
+    },
+    getIdentifier(url: string): string {
+      return url.replace(/(https|http):\/\//gi, '').split('/')[1];
+    },
+    getOverviewUrl(url: string): string {
+      return `${Animetoast.domain}/${Animetoast.sync.getIdentifier(url)}`;
+    },
+    getEpisode(url: string): number {
+      const matching = $('.current-link').text().match(/(\d+)/i);
+      return matching ? parseEpisode(matching) : NaN;
+    },
+    nextEpUrl(url: string): string | undefined {
+      return $('.current-link').next().attr('href');
+    },
+    uiSelector(selector: string) {
+      $('.title-info > .entry-title.light-title').first().after(j.html(selector));
+    },
+  },
+  overview: {
+    getTitle(url: string): string {
+      return Animetoast.sync.getTitle(url);
+    },
+    getIdentifier(url: string): string {
+      return Animetoast.sync.getIdentifier(url);
+    },
+    uiSelector(selector: string) {
+      Animetoast.sync.uiSelector!(selector);
+    },
+    list: {
+      offsetHandler: true,
+      elementsSelector(): JQuery<HTMLElement> {
+        return $('.tab-content > div > a');
+      },
+      elementUrl(selector: JQuery<HTMLElement>): string {
+        return selector.attr('href')!;
+      },
+      elementEp(selector: JQuery<HTMLElement>): number {
+        return parseEpisode(selector);
```

Comment:
Same as above. And it is wrong either way. Here probably selector.text() is the way to go